src/Microsoft.AspNet.Razor/Generator/GeneratedTagHelperContext.cs

```diff
@@ -150,5 +152,15 @@ public GeneratedTagHelperContext()
         /// Contains information about in-scope tag helpers, HTML attributes, and the tag helpers' output.
         /// </remarks>
         public string ExecutionContextTypeName { get; set; }
+
+        /// <summary>
+        /// The name of the property in HtmlHelper used to encode html.
```

Comment:
better if `HtmlHelperPropertyName` remains: `The name of the <c>IHtmlHelper</c> property ...`.  Otherwise `The name of the property used to encode HTML.  Expected to be of type <c>IHtmlEncoder</c>.`